pkg/apis/hive/v1/agent/platform.go

```diff
@@ -0,0 +1,68 @@
+package agent
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Platform stores cluster configuration for an agent based install where hosts are booted and registered, ready
+// to be assigned and used in a cluster.
+type Platform struct {
+	// Cluster name
+	Name string `json:"name"`
```

Comment:
Looks like we can drop this one, this would be ClusterDeployment.Spec.ClusterName.

Reply:
Agree